angular/core/services/session.coffee

```diff
@@ -24,5 +24,11 @@ angular.module('loomioApp').factory 'Session', ($rootScope, Records, AppConfig)
     $rootScope.$broadcast 'loggedIn', @user()
     @user()
 
+  homePath: ->
+    switch @user().groups().length
+      when 0 then '/explore'
+      when 1 then LmoUrlService.group(@user().groups()[0])
```

Comment:
A thought for a future iteration:

I think we should be smarter about what kind of multiple groups you're in. IE, if you're in Loomio Community and Workers, does it make sense to land you on the pretty parent group page, rather than sending you to a dashboard?